_posts/2023-09-08-training-series.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "Learning SQL with Trino from the experts"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: 
+show_pagenav: false
+---
+
+Do you think SQL stands for Silly Quacking Leopards and write all your code in
+Java, Python, or Go? But you need to get data out of object storage in the cloud
+and some relational database at the same time? You should look at Trino and
+learning about SQL.
+
+Or do you know that ins and outs of joins, window functions, and your SQL
+queries are counted by the pages and not lines? You are the expert on SQL in
+your database team. You should also look at Trino and SQL.
+
+Luckily for you all we got the right SQL training coming up for everyone in our
+upcoming series with the founders of the Trino project and SQL experts Martin
+Traverso, Dain Sundstrom, and David Phillips, and myself as host and co-trainer.
+
+<!--more-->
+
+In the SQL training series we start with the basics around Trino, so you we are
+all on the same page. You will know the benefits of connecting [many data
+sources]({{site_url}}/ecosystem.html#data-sources) to Trino, and using
+[different clients]({{site_url}}/ecosystem.html#clients), but always using the
+same powerful SQL. And for the SQL pros, you learn about catalogs and queries
+that go across data sources.
+
+Then we glance at the really basic SQL foundations, since there are literally
+hundreds of books, videos, and training course around. All of them teach you
+things like `SELECT` statements, and `WHERE` clauses, and unravel the confusions
+around `LEFT OUTER JOIN` and the like.
+
+And thats where we are getting to the interesting stuff. Following is a list of
+some of stuff we will cover:
```

Comment:
```suggestion
some of the topics we will cover:
```
And we _are_ using the future tense here, so we should make it consistent one way or the other.

Reply:
Done